src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -38,8 +40,18 @@
     public static final String VALID_TAG_FRIEND = "friend";
     public static final String VALID_MODULE_CODE_CS50 = "CS50";
     public static final String VALID_MODULE_CODE_CS1010S = "CS1010S";
+    public static final String VALID_MODULE_CODE_CS2103 = "CS2103";
     public static final String VALID_MODULE_NAME_CS50 = "Introduction to Computer Science";
     public static final String VALID_MODULE_NAME_CS1010S = "Programming Methodology";
+    public static final String VALID_MODULE_NAME_CS2103 = "Software Engineering";
+
```

Comment:
Perhaps it would be better to use MODULE_CODE and MODULE_NAME instead? I think it avoids confusion and keeps it consistent.

Reply:
alright!